web/src/components/MediaControl/MediaControl.scss

```diff
@@ -18,10 +19,24 @@
 }
 
 .media-control {
-  font-size: 3.5rem;
   color: general.$controls-color;
   padding-left: 1.5rem;
   padding-right: 1.5rem;
+
+
+@media (max-width: 365px) {
+  font-size: 1.5rem;
+  padding-left: 1.25rem;
+  padding-right: 1.25rem;
+}
+
+@media (min-width: 365px) and (max-width: 425px) {
+  font-size: 2.5rem;
+}
+
+@media (min-width: 425px) {
+  font-size: 3.5rem;
+}
 }
```

Comment:
is this correct? if it isn't the indentation needs to be corrected

Reply:
Only problem was a tabindex, fixed in recent commit